client/app/components/packages/pas-form/applicant-team.hbs

```diff
@@ -54,5 +54,13 @@
         </div>
       </div>
     </div>
+    {{#unless @pasForm.hasApplicants}}
+      <span
+        class="form-error is-visible"
+        data-test-validation-message="has-applicants"
+      >
+        One or more applicant team members is required.
+      </span>
+    {{/unless}}
```

Comment:
I think this works nicely, but it seems like we're missing out on the features of changeset validations. Is it possible to validate the `.length` property of `applications`?

To me, this is a validation and should leverage the validator features we have. I'm happy to take a look! I've done a lot of research on ways to use changeset validations on related models.

Reply:
That would be sweet, if you could do that! Maybe it's possible to use a custom validator. 
We would have to spike test if the oldValue and newValue parameters reflect related models
```
![image](https://user-images.githubusercontent.com/3311663/87803412-479cb500-c807-11ea-90e8-a233b5f84d8f.png)

```

Reply:
Though would this be muddying the focus of the validators? Should we keep them focused to just field-specific validations? We've made an effort to move the changesets off of the models and into their own component. Would this potentially couple the changesets and models again?

Reply:
> Though would this be muddying the focus of the validators? Should we keep them focused to just field-specific validations? We've made an effort to move the changesets off of the models and into their own component. Would this potentially couple the changesets and models again?

It is field-specific: the model has a field called `applicants` and we want to validate that it has at least one.

I don't quite follow your last point... changesets are really just a proxy of the model itself, but the proxy intercepts changes to it. When were changesets coupled to the model? I think it should behave just as it does with any other field. There's an issue in ember changeset validations addon about this, I'll just have to dig it up.

Reply:
Oops sorry, I misspoke the changesets used to be created on the edit Packages::_Form::Edit component. I thought the changesets used to be on the package model itself. Scratch the model thing.

Okay, sounds good, thanks!